jina/resources/k8s/template/service.yml

```diff
@@ -7,10 +7,6 @@ metadata:
     app: {target}
 spec:
   type: {type}
-  ports:
-    - port: {port}
-      targetPort: {port}
-      name: port
-      protocol: TCP
+  ports: {service-ports-section}
```

Comment:
why this change?

Reply:
for the service to work properly with multiple ports, we need to put of them as items here, not just 1 item